data/free worlds middle.txt

```diff
@@ -1411,13 +1416,14 @@ mission "FW Bloodsea 1.1"
 					goto invade
 			label port
 			`	"Yes," he says, "except that I can almost guarantee that when we show up on their doorstep with everything they asked for, some local warlord will attack our ships, steal our supplies, enslave our workers, and build himself a nice cozy island fortress."`
-				goto attack
-			label invade
-			`	"That's certainly the simpler option," he says, "especially because I can almost guarantee that when we show up on their doorstep with everything they asked for, some local warlord will attack our ships, steal our supplies, enslave our workers, and build himself a nice cozy island fortress."`
-				goto attack
-			label attack
 			`	"So, we attack them?" you ask.`
```

Comment:
Should this text, as well as `label attack` be moved as well?

Reply:
What do you mean? I've removed `label attack` from this conversation.

Reply:
I think you should *retain* `label attack` as well, and move this and its label to before the new position of "You're on the Council now".

As-is, I believe what will happen, if you select "port" option here, because of fallthrough, if the player never suppressed Bloodsea:

> "Well, building a spaceport is probably cheaper than fighting a major battle."`
>
> "Yes," he says, "except that I can almost guarantee that when we show up on their doorstep with everything they asked for, some local warlord will attack our ships, steal our supplies, enslave our workers, and build himself a nice cozy island fortress."
>
> "So, we attack them?" you ask.
>
> 	"That's certainly the simpler option," he says, "especially because I can almost guarantee that when we show up on their doorstep with everything they asked for, some local warlord will attack our ships, steal our supplies, enslave our workers, and build himself a nice cozy island fortress. Tomek may have been punished for disobeying the Senate's orders, but we're in a different situation now. Our numbers have swelled because of our victories and new territory, and we now have the Dreadnoughts at our disposal, so occupying Bloodsea is now a viable option."`

Mentioning the "cozy island fortress" twice with almost the same wording each time after the player asks about attacking them, seems a bit too repetitive of Alondo.

Reply:
Ah, hadn't noticed the double mention of the fortresses. Good catch.